content/english/python-discord-translate/creating-discord-bot/writing-the-bot/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Writing a simple bot"
+draft: false
+weight: 2
+---
+
+Nuvi now has a registered bot identity. Next: make it actually respond. With your project folder, virtual environment, and starter file ready, you’ll build a **minimal bot foundation**. This first version just proves the connection works. We also turn on one permission (`message_content`) now so later pages can add auto‑translation without revisiting setup.
+
+In this part, you will:
+
+1. Add `discord.py` to `requirements.txt` and install it (VS Code UI — no terminal typing required)
+2. Write a basic bot that responds to a slash command `/hello`
+3. Enable the message content intent (needed later for auto‑translation)
+4. Run and test the bot
+
+{{< alert theme="info" >}}
+<img src="../../media/NF_mascot.jpg" alt="Nuvi mascot" width="85" style="float:right;margin:0 0 8px 12px;" />
+<strong>Nuvi says:</strong> Your first goal is just to see the bot come online—translation comes later. Small wins keep momentum!
+{{< /alert >}}
+
+---
+
+## 1. Add and Install the discord.py Library (Using requirements.txt + VS Code UI)
+We’ll declare the dependency first, then let VS Code install it into your virtual environment.
+
+### 1.a Add it to requirements.txt
+1. Open `requirements.txt` in your project.
+1. Add this line (the file can just contain this for now):
+```
+discord.py
+```
+1. Save the file
```

Comment:
Fix the numbering:
<img width="1164" height="127" alt="Screenshot 2025-09-17 at 4 05 56 PM" src="https://github.com/user-attachments/assets/58d6d19f-7262-4e4b-985f-d58ee0db1e12" />